python/utils/worker.py

```diff
@@ -488,8 +521,65 @@ def run(self):
                 self.processing_result = self.processor.process_transactions(
                     self.transactions, start_version, end_version
                 )
-
+                num_of_transactions = str(end_version - start_version + 1)
+                processor_name = self.processor.name()
+                start_version_str = str(start_version)
+                end_version_str = str(end_version)
+                processing_duration_in_secs = format(
+                    self.processing_result.processing_duration_in_secs, ".8f"
+                )
+                db_insertion_duration_in_secs = format(
+                    self.processing_result.db_insertion_duration_in_secs, ".8f"
+                )
+                duration_in_secs = format(
+                    self.processing_result.processing_duration_in_secs
+                    + self.processing_result.db_insertion_duration_in_secs,
+                    ".8f",
+                )
+                size_in_bytes = str(sum(obj.ByteSize() for obj in self.transactions))
```

Comment:
I think we avoided doing it this way to avoid adding latency to processors. Since we have `response.ByteSize()` already, we can pass it as a value in the channel item, which can be consumed downstream

Reply:
done.